node/node_handler.go

```diff
@@ -55,95 +55,130 @@ func (node *Node) processSkippedMsgTypeByteValue(
 	}
 }
 
-// HandleMessage parses the message and dispatch the actions.
-func (node *Node) HandleMessage(content []byte, sender libp2p_peer.ID) {
-	msgCategory, err := proto.GetMessageCategory(content)
-	if err != nil {
-		utils.Logger().Error().
-			Err(err).
-			Msg("HandleMessage get message category failed")
-		return
-	}
-	msgType, err := proto.GetMessageType(content)
-	if err != nil {
-		utils.Logger().Error().
-			Err(err).
-			Msg("HandleMessage get message type failed")
-		return
-	}
+func (node *Node) handleClientMessage(
+	ctx context.Context, isConsensusBound bool, msgPayload []byte,
+) error {
 
-	msgPayload, err := proto.GetMessagePayload(content)
-	if err != nil {
-		utils.Logger().Error().
-			Err(err).
-			Msg("HandleMessage get message payload failed")
-		return
-	}
+	return nil
```

Comment:
return nil?

Reply:
means success, errors now captured in values, before these APIs didn't return errors

Reply:
but actualy ya - now I remember, I forgot, this one should be explicit error since something should have been handled explicitly but then I'd need to add returns more and I punted.

Reply:
I mean this func just directly returns nil. The below logic is never executed..

Reply:
Ya it’s leftover code , I just return nil to not have to comment out , it’s handled else where

Reply:
I also couldn't find the reference on this func "handleClientMessage"

Reply:
Seems duplicate to HandleNodeMessage

Reply:
It will make sense , look at node.go only first